spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -447,6 +447,22 @@
       end
     end
 
+    context 'with a user that accepted the rules of use more than 6 years ago' do
+      let(:rules_of_use_updated_at) { 7.years.ago }
+      let(:accepted_terms_at) { 6.years.ago - 1.day }
+      let(:user) { create(:user, :signed_up, accepted_terms_at: accepted_terms_at) }
+
+      before do
+        allow(IdentityConfig.store).to receive(:rules_of_use_updated_at).
```

Comment:
This should be... ?

```suggestion
        allow(IdentityConfig.store).to receive(:rules_of_use_horizon_years).
```

Reply:
This is here to override the original value that `rules_of_use_updated_at` may have had. If the rules of use was updated within the last 6 years + 1 day the user will have to re-accept no matter what. To actually test the feature we need to push the last update timestamp back further than when the user accepted.

Reply:
I think tests like this are clearer when we override the config values, I never remember to check config value defaults when doing tests, so it's nice to have some sort of explicit override

Reply:
I'll add some explicit overrides